net/socket/udp_socket_unittest.cc

```diff
@@ -740,9 +745,9 @@ TEST_F(UDPSocketTest, JoinMulticastGroup) {
   EXPECT_THAT(socket.JoinGroup(group_ip), IsOk());
   // Joining group multiple times.
   EXPECT_NE(OK, socket.JoinGroup(group_ip));
-  EXPECT_THAT(socket.LeaveGroup(group_ip), IsOk());
-  // Leaving group multiple times.
-  EXPECT_NE(OK, socket.LeaveGroup(group_ip));
+  // EXPECT_THAT(socket.LeaveGroup(group_ip), IsOk());
```

Comment:
If this fails on Starboard, then please leave a comment here with a tracking bug.

Reply:
Oops, I accidentally commented these few lines out, going to change it back. But this test case is disabled anyway. Like what I comment in the commit message, these test cases fail at trunk due to Starboard implementation, and after this PR, we keep the same behavior.